client/components/tinymce/plugins/insert-menu/menu-items.jsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+
+import Gridicon from 'components/gridicon';
+import SocialLogo from 'components/social-logo';
+import i18n from 'lib/mixins/i18n';
+
+const GridiconButton = ( { icon, label } ) => (
+	<div>
+		<Gridicon className="wpcom-insert-menu__menu-icon" icon={ icon } />
+		<span className="wpcom-insert-menu__menu-label">{ label }</span>
+	</div>
+);
+
+const SocialLogoButton = ( { icon, label } ) => (
```

Comment:
was used, then wasn't used in this current PR - it was put in for the Instagram widget and I just didn't take it out.